docs/source/models/templates/concept_model_template.rst

```diff
@@ -72,188 +79,242 @@ Concept model template
 
 .. _{YOUR_MODEL_SHORT_NAME}1.0:
 
-1.0 Background
-++++++++++++++
-
-
-.. _{YOUR_MODEL_SHORT_NAME}1.1:
-
-1.1 Project overview
---------------------
-
-
-
-.. _{YOUR_MODEL_SHORT_NAME}1.2:
-
-1.2 Literature review
----------------------
+1.0 Project overview
+++++++++++++++++++++
+Here you can provide an introduction and overview to the project. Information
+in this section might include the following: 
 
+* Brief background of topic
+* Purpose or aims
+* Funder information
+* Examples of similar analyses
 
 .. _{YOUR_MODEL_SHORT_NAME}2.0:
 
-2.0 Modeling aims and objectives
-++++++++++++++++++++++++++++++++
+2.0 Simulation Design
+++++++++++++++++++++++
 
+This section is intended as the main body of documentation of your simulation. Information
+in this section might include the following: 
 
-.. _{YOUR_MODEL_SHORT_NAME}3.0:
+* Concept model diagram 
+* Links to relevant components of diagram
+* Scenario descriptions
+* Default specifications 
+* Demographics
+* Location description
+* Effect size
 
-3.0 Causal framework
-++++++++++++++++++++
+.. _{YOUR_MODEL_SHORT_NAME}2.1:
 
-.. _{YOUR_MODEL_SHORT_NAME}3.1:
-
-3.1 Causal diagram
-------------------
- 
- .. note::
-    link to DAGs page
-    use round circles with DAGs
-
-**Outcome (O)**:
+2.1 Concept model diagram 
+-------------------------
 
+.. _{YOUR_MODEL_SHORT_NAME}2.2:
 
+2.2 Default specifications 
+--------------------------
 
-**Most proximal determinant/exposure (E)**:
+The below table is intended to outline the default specifications of your simulation. 
+Included in the table is a column of parameter definitions. Please delete this column as you 
+fill out the table. 
+
+.. list-table:: Default simulation specifications
+  :header-rows: 1
+
+  * - Parameter
+    - Definition
+    - Value
+    - Note
+  * - Location(s)
+    - Target/simulated region & GBD ID 
+    - e.g. Ethiopia (ID: 179)
+    -
+  * - Number of draws
+    - Desired number of draws that a given simulation is to be run for. (Generally, this should be a number between 1 and 1,000.)
+    - e.g. 50 draws 
+    - Read more about draws in Vivarium :ref:`here <vivarium_best_practices_monte_carlo_uncertainty>`.
+  * - Population size per draw
+    - Desired simulated population size per draw for a given simulation. 
+    - e.g. 100,000 simulants
+    - Read more about how to determine a reasonable population size for your simulation 
+      :ref:`here <vivarium_best_practices_monte_carlo_uncertainty>. Note that engineers will
+      decide how many random seeds to use to achieve the specified population size per draw.`.
```

Comment:
```suggestion
      :ref:`here <vivarium_best_practices_monte_carlo_uncertainty>`. Note that engineers will
      decide how many random seeds to use to achieve the specified population size per draw.
```
I think a misplaced backtick got introduced in commit 07c66ad4c59bbad62613d11b9c54879935b2a72f, which is probably what's causing the build to fail.